tests/functional/conftest.py

```diff
@@ -44,13 +57,27 @@ def wait_for_maximum_list_length(
     ) -> None:
         """Wait for the list() length to be no greater than expected maximum or fail
         test if timeout is exceeded"""
-        for _ in range(max_iterations):
-            if len(rest_manager.list()) <= max_length:
+        logging.info(f"Checking {description!r} has no more than {max_length} items")
```

Comment:
Can we then make this function name shorter as we get log lines quite a bit longer?
```
2022-01-04 05:49:51.067 [    INFO] (conftest.py:wait_for_maximum_list_length:L60) Checking 'projects' has no more than 0 items
```

Reply:
Sure. I changed it to `wait_for_list_size`